src/widgets/immunizations/immunizations.component.tsx

```diff
@@ -1,21 +1,20 @@
-import React, { useEffect } from "react";
-import { Switch, Route, useRouteMatch } from "react-router-dom";
+import React from "react";
+import { Route, BrowserRouter, Switch } from "react-router-dom";
 import ImmunizationsDetailedSummary from "./immunizations-detailed-summary.component";
-import { defineConfigSchema } from "@openmrs/esm-config";
 import withConfig from "../../with-config";
 import { ConfigObject } from "../../config-schema";
 
 function Immunizations(props: ImmunizationsProps) {
-  const match = useRouteMatch();
-
   return (
-    <Switch>
-      <Route exact path={match.path}>
-        <ImmunizationsDetailedSummary
-          immunizationsConfig={props.config.immunizationsConfig}
-        />
-      </Route>
-    </Switch>
+    <BrowserRouter basename={window["getOpenmrsSpaBase"]()}>
```

Comment:
Its a single route definition. Do we need this (the question applies to all the widgets using this pattern)?

Is this about using `useRouteMatch` / using the `patientUuid` parameter in the path / using some `<Link>` inside?

Personally, I think the `patientUuid` should be transported via props to the extension slot (defined in patient-chart).